src/pages/ReportingRateSummary.js

```diff
@@ -17,22 +17,20 @@ export default class ReportingRateSummary extends React.Component {
         return (
             <div>
                 <SectionHeadline
-                    label={i18n.t('Resource')}
-                    showBackButton={!!props.reportHtml}
-                    onBackClick={props.unsetReportData}
+                    label={i18n.t('Reporting rate summary')}
                     systemVersion={props.d2.system.version}
                     sectionKey={props.sectionKey}
                 />
                 <Paper style={styles.container}>
                     <Form
-                        showForm={!props.reportHtml}
+                        showForm={true}
```

Comment:
To me this means that `showForm` should not be required and default to true in the Form component.

Reply:
Aha yes, I actually did that for another component but forgot it here.